css-animations-2/Overview.bs

```diff
@@ -247,133 +247,133 @@ from its <a>owning element</a> always causes it to enter (or remain) in the
-
-                Note, since multiple keyframe blocks may specify the same
-                |keyframe offset|,
-                and since this algorithm iterates over these blocks in reverse,
-                this implies that if any properties are encountered that
-                have already added at this same |keyframe offset|,
-                they should be skipped.
-
-     1.   Add each property name that was added to |keyframe|
-          to |animated properties|.
-
-1.   If there is no keyframe in |keyframes| with offset 0,
-     or if amongst the keyframes in |keyframes| with offset 0
-     not all of the properties in |animated properties| are present,
-
-     1.   Let |initial keyframe| be the [=keyframe=] in |keyframes|
-          with offset 0, timing function |default timing function|
-          and composite |default composite|.
-
-          If there is no such keyframe,
-          let |initial keyframe| be a new empty keyframe with offset 0,
-          timing function |default timing function|,
-          composite |default composite,
-          and add it to |keyframes| after the last keyframe with offset 0.
-
-     1.   For each property in |animated properties| that is not present
-          in some other keyframe with offset 0,
-          add the [=computed value=] of that property for |element|
-          to the keyframe.
-
-1.   Similarly, if there is no keyframe in |keyframes| with offset 1,
-     or if amongst the keyframes in |keyframes| with offset 1
-     not all of the properties in |animated properties| are present,
-
-     1.   Let |final keyframe| be the [=keyframe=] in |keyframes|
-          with offset 1, timing function |default timing function|
-          and composite |default composite|.
-
-          If there is no such keyframe,
-          let |final keyframe| be a new empty keyframe with offset 1,
-          timing function |default timing function|
-          and composite |default composite|,
-          and add it to |keyframes| after the last keyframe with offset 1.
-
-     1.   For each property in |animated properties| that is not present
-          in some other keyframe with offset 1,
-          add the [=computed value=] of that property for |element|
-          to the keyframe.
-
-Issue: The above procedure requires iterating over keyframe blocks in reverse.
-It could be rewritten so this is not required but that will likely change
-the behavior for some edge cases.
-We should verify what current implementations do and possible remove the
-requirement to iterate in reverse.
+For each animation effect defined by the <var>N</var>th item
+in the [=coordinated value list=] of the 'animation-*' properties
+on target (pseudo-)element |element|,
+its associated [=keyframes=] are generated as follows:
+
+1.  <strong>Set Defaults:</strong>
+
+    *   Let |default timing function| be the corresponding value
+        of 'animation-timing-function'.
+
+    *   Let |default composite| be ''replace''.
```

Comment:
This should come from the `animation-composition` property.

Reply:
Done. (Added a new commit for that, since this was the original state.)